osu.Game/Localisation/Language.cs

```diff
@@ -110,6 +110,11 @@ public enum Language
         // zh_hk,
 
         [Description(@"繁體中文（台灣）")]
-        zh_hant
+        zh_hant,
+
+#if DEBUG
+        [Description(@"Debug (show raw keys)")]
+        DebugLocalisation
```

Comment:
Probably best to pin this to a value since we're likely going to be adding more languages.

Reply:
I can't find anywhere we're referencing by `int`, do you still think this is important?

Reply:
Is it referenced by text in the config...?

Reply:
Looks like it is. Fine to leave as is then.

Reply:
Yep, that's how config managers work:

![JetBrains Rider-EAP 2022-04-19 at 08 45 23](https://user-images.githubusercontent.com/191335/163962871-756b27ee-73d4-4bda-bc56-a3fa4c0286a2.png)

We decided on using the enum representation in a distant past.